node/forest_libp2p/src/behaviour.rs

```diff
@@ -110,6 +120,30 @@ impl NetworkBehaviourEventProcess<KademliaEvent> for ForestBehaviour {
     }
 }
 
+impl NetworkBehaviourEventProcess<BitswapEvent> for ForestBehaviour {
+    fn inject_event(&mut self, event: BitswapEvent) {
+        match event {
+            BitswapEvent::ReceivedBlock(peer_id, cid, data) => {
+                let cid = cid.to_bytes();
+                let cid: Cid = Cid::from_raw_cid(cid.as_slice()).unwrap();
+                self.events.push(ForestBehaviourEvent::BitswapReceivedBlock(
+                    peer_id, cid, data,
+                ));
+            }
+            BitswapEvent::ReceivedWant(peer_id, cid, _priority) => {
+                let cid = cid.to_bytes();
+                let cid: Cid = Cid::from_raw_cid(cid.as_slice()).unwrap();
```

Comment:
don't unwrap, again can't guarantee this will succeed

Reply:
ah, sorry. left these in by accident